zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1519,6 +1519,42 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /process-instances/batch-operations/cancellation:
+    post:
+      tags:
+        - Process instance
+      operationId: cancelProcessInstancesBatchOperation
+      summary: Create a batch operation to cancel process instances
+      description: |
+        Cancels multiple running process instances.
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ProcessInstanceFilter"
+      responses:
+        "201":
+          description: The key of the batch operation.
```

```suggestion
          description: The process instances were successfully cancelled.
```

🔧 This description should describe the operation status. I may not have described it perfectly, but hopefully the suggestion gives you enough to go on to describe it more accurately.

---

But this is not true, the 201 just says that zeebe got the request. But it's async, so at this point, nothing is canceled.

---

If that's the case it seems like [202](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status/202) is a more appropriate status code.

---

Yes, that's better. :)

---

Also changed description to: "The batch operation was successfully started."

---

💭 I believe sending the request just means it has been sent/submitted/created, but not necessarily `started`

---

Then maybe "The batch operation request was accepted"? (What also the HTTP Code says)